### PR TITLE
new icon: windows11 (original, original-wordmark)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes https://github.com/devicons/devicon/issues/800**

## Link to prove your SVG is correct and up-to-date.

https://en.wikipedia.org/wiki/Windows_11#/media/File:Windows_11_logo.svg

### CODE REVIEW

**Comment by PantheraRed** [comment]
Opening a new pr in reference to https://github.com/devicons/devicon/pull/822.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please fix the issues mentioned by the `check-bot` workflow.
Let us know if you need any help with that. 🙂

**Comment by PantheraRed** [comment]
Sorry @Panquesito7. I completely missed this and placed windows11 after windows8.
image

File: devicon.json [hunk 1]
```diff
@@ -9719,6 +9719,25 @@
...
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#0078d4",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:9739]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```

**Review by Panquesito7 — APPROVED** [review]
Awesome. Thanks! 🚀

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark: 
Thank you for another great contribution @PantheraRed :rocket:

**Pull request merged**